packages/generator/src/generator.ts

```diff
@@ -16,6 +16,88 @@ import {readdirRecursive} from "./utils/readdir-recursive"
 import prettier from "prettier"
 const debug = require("debug")("blitz:generator")
 
+export const addCustomTemplatesBlitzConfig = (
+  customTemplatesPath: string,
+  isTypeScript: boolean,
+) => {
+  const blitzServer = isTypeScript ? "app/blitz-server.ts" : "app/blitz-server.js"
+  const blitzServerPath = require("path").join(process.cwd(), blitzServer)
+  const userConfigModuleSource = fs.readFileSync(blitzServerPath, {encoding: "utf-8"})
+  const userConfigModule = j(userConfigModuleSource, {parser: customTsParser})
+  const program = userConfigModule.get()
+  const cliConfigDeclaration = userConfigModule
+    .find(j.ExportNamedDeclaration, {
+      declaration: {
+        type: "VariableDeclaration",
```

Comment:
Couldn't we use esbuild or something to eval this file and extract default instead?

Reply:
Ah right 🤦‍♀️ ok, ignore my question